Formula/ale.rb

```diff
@@ -0,0 +1,38 @@
+class Ale < Formula
+  desc "The Arcade Learning Environment (ALE) -- a platform for AI research"
+  homepage "https://github.com/mgbellemare/Arcade-Learning-Environment"
+  url "https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.5.2.tar.gz"
+  sha256 "bc158b4fb8edc5034f8f8b4ca1a90d48590f516b33f9edec325fce8762418069"
+  head "https://github.com/mgbellemare/Arcade-Learning-Environment.git"
+
+  depends_on "cmake" => :build
+  depends_on "sdl"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+
+    if build.with? "python"
```

Comment:
You need to add a `without-python` option or this is always false except on snow_leopard

Reply:
That said, there's no reason to make it optional at all.